action.php

```diff
@@ -31,7 +31,7 @@ public function register(Doku_Event_Handler $controller) {
 
         $controller->register_hook('AJAX_CALL_UNKNOWN', 'BEFORE', $this, 'handle_ajax_call_unknown');
         $controller->register_hook('MULTIORPHAN_INSTRUCTION_LINKED', 'BEFORE', $this, 'handle_unknown_instructions');
-   
+        $controller->register_hook('DOKUWIKI_STARTED', 'AFTER', $this, 'extend_JSINFO');
```

Comment:
It will always populate the JSINFO though the plugin might not be loaded and executed. Do you have an idea how to solve this differently?